src/App.js

```diff
@@ -64,7 +63,7 @@ export default function App() {
   const currPage = getCurrPage(searchParams, totalPages);
 
   return (
-    <div className="overflow-y-hidden overflow-x-hidden">
+    <div className="overflow-y-clip overflow-x-clip">
```

Comment:
what does clip do?

Reply:
This was one of Alex's changes "the clip keyword also forbids all scrolling, including programmatic scrolling"